### PR TITLE
Improved the example commands in running-on-k8s document.

### PR DESCRIPTION
**Pull request by lins05**
## What changes were proposed in this pull request?

Improved the example commands in running-on-k8s document so it's easier to copy & paste. Also fixed the name of the example jar.

### CODE REVIEW

**Review by ash211 — COMMENTED** [review]
Looks like a good change to me, thanks @lins05 !

There are several other places in this file where a spark-submit command is split across multiple files without the `\` at the end of lines.  Would you please add the `\` to those also and then we can merge?

Welcome to the Spark on Kubernetes efforts @lins05 !

**Comment by lins05** [comment]
@ash211 Thanks, updated!

File: docs/running-on-kubernetes.md [hunk 1]
```diff
@@ -75,54 +75,54 @@ examples of providing application dependencies.
...
-      http://example.com:8080/applications/sparkpluggable/app.jar
+    bin/spark-submit \
+      --deploy-mode cluster \
+      --class com.example.applications.PluggableApplication \
+      --master k8s://https://192.168.99.100 \
+      --kubernetes-namespace default \
+      --conf spark.jars=file:///opt/spark-plugins/app-plugin.jar \
+      --conf spark.kubernetes.driver.docker.image=registry-host:5000/spark-driver-custom:latest \
+      --conf spark.kubernetes.executor.docker.image=registry-host:5000/spark-executor:latest \
+      http://example.com:8080/applications/sparkpluggable/app.jar \
```
Comment (ash211) [inline docs/running-on-kubernetes.md]: one extra slash on the last line here

Comment (lins05) [reply docs/running-on-kubernetes.md]: Good catch, removed.

**Comment by ash211** [comment]
Looks good -- merging!

Thanks again for the contribution @lins05 !  Have you been able to get this code working based on these docs?

**Comment by lins05** [comment]
Yep. I have setup a minikube cluster and sucessfully run the spark pi example in cluster mode, and it's very straight forward by following the doc. It did took me some time to build the spark dist files and the docker images though.

**Comment by ash211** [comment]
Yep they build pretty slowly -- Spark is a rather large project.  If you have issues or suggestions, please file issues in this project and provide feedback feedback.  Glad you were able to get it working.

**Comment by foxish** [comment]
@lins05 Have you signed the Apache ICLA/CCLA? @ash211 I think we should make sure that contributors have one of those signed especially if they contribute code.

**Comment by lins05** [comment]
@foxish Not yet, but I'd be happy to, so please tell me how to proceed. Also FWIW I've contributed code to both mesos and spark, but neither of them asked me to sign the CLA.

**Comment by foxish** [comment]
I know that @iyanuobidele signed one, but that was the CCLA IIRC.

**Comment by iyanuobidele** [comment]
Indeed, it was the CCLA.

For me, we already had one signed with Apache, so I only had to add my name in the mix.

@lins05 if you're part of a company, then you might want to get legal and ... to look into this [document](https://www.apache.org/licenses/cla-corporate.txt). Otherwise, you need an ICLA, which I believe is more straightforward. See [here](https://www.apache.org/licenses/icla.txt)

**Pull request merged**